### DISCUSSION TITLE
Show failed speedtests

### DISCUSSION THREAD

**Discussion by demianovics** [alexjustesen/speedtest-tracker#303] [Ideas]
Just like i could with henrywhitaker3/speedest-tracker, i'd like to see when speedtests fail. 

But so far only manually queued speedtests did fail without notification or listing in the results. So maybe fails with automatic runs will show up.

**Comment by alexjustesen** [comment]
Already got this one planned for an upcoming release. https://github.com/alexjustesen/speedtest-tracker/issues/143
